### ISSUE TITLE
examples for YOLOv5

### ISSUE THREAD

**Issue by CangHaiQingYue** [quic/aimet#2809]
Do these have any code examples for YOLOv5 Qat? I encountered many problems during the implementation process and tried to solve some of them. The current problem is that during the fine-tune stage, the indicators are particularly low. It seems like training started  without pre-trained model.
Thanks！

**Comment by quic-mangal** [comment]
@CangHaiQingYue, did you evaluate the model after computing the encodings?

**Comment by CangHaiQingYue** [comment]
> @CangHaiQingYue, did you evaluate the model after computing the encodings?

yes. I Ihave fixed this bug, and the biggest issue now is that the training time for per-channel is particularly slow. 
per-tensor is 7min， per-channel is 1h30min...

**Comment by quic-mangal** [comment]
Which quant-scheme are you using?

**Comment by shuyuan-wang** [comment]
> Which quant-scheme are you using?

Hi, did your team try yolov5? I'm using percentile quant scheme and set percentile value as 99.996. The PTQ result is not good. I'm training my own dataset with three labels.

**Comment by shuyuan-wang** [comment]
> Which quant-scheme are you using?

Hi, did your team try yolov5? I'm using percentile quant scheme and set percentile value as 99.996. I'm not sure why the evaluation shows all zero. after I compute the encodings, I saved the quant sim model and load them in val.py file.
